[DAS] Add test (#1006)
trueagi-io/hyperon-experimental: docs/das_setup.md

Starting MongoDB service...
MongoDB has started successfully on port 28000 at localhost, operating under the server user arturgontijo.
```

3. Load Databases with `das_kb.metta` content:
```
# We need the metta file's absolute path, moving it to /tmp for convinience
cp lib/tests/das_kb.metta /tmp

# Then
das-cli metta load /tmp/das_kb.metta
# Or
python3 das-cli/src/das_cli.py metta load /tmp/das_kb.metta

>>>
das-cli-mongodb-28000 is running on port 28000
das-cli-redis-29000 is running on port 29000
Loading metta file /tmp/das_kb.metta...
Connecting to Redis at localhost:29000
Connecting to MongoDB at localhost:28000
Done.
```

4. Start Attention Broker service
```
das-cli ab start
# Or
python3 das-cli/src/das_cli.py ab start

>>>
Starting Attention Broker service...
Attention Broker started on port 37007
```

5. Start Query Broker service
```
das-cli qa start
# Or
python3 das-cli/src/das_cli.py qa start

>>>
# Chose a port range (using 52000:52999)
Enter port range (e.g., 3000:3010): 52000:52999
Starting Query Agent service...
Query Agent started on port 40002
```

6. Running DAS tests
```

> # Be sure the REPL is build (from repo's root)
> cargo b -r
> 
> # Run it passing a metta file
> # NOTE: this run will freeze for ~20s as client tries to get all available services from peers
> ./target/release/metta-repl lib/tests/das.metta

Reviewer: Cargo run can be used, may be it is simpler than typing full path to the built binary:
```suggestion
# Run it passing a metta file (from repo's root)
# NOTE: this run will freeze for ~20s as client tries to get all available services from peers
cargo run -r --bin metta-repl lib/tests/das.metta
```
